packages/la-boite/src/repos/local_git_repos.spec.ts

```diff
@@ -30,8 +30,8 @@ describe(LocalGitRepos.name, () => {
   })
 
   describe('waiting for a repo to become idle', () => {
-    it('resolves immediately if the repo is already idle', () => {
-      promiseThat(repos.waitUntilIdle('a-repo'), fulfilled())
+    it('resolves immediately if the repo is already idle', async () => {
+      await promiseThat(repos.waitUntilIdle('a-repo'), fulfilled())
```

Comment:
@mattwynne be careful with promise test helpers. I've been bitten by similar things in `chai-as-promised`, they still are promise-based.

Reply:
Got it, thanks